blog/urls.py

```diff
@@ -0,0 +1,11 @@
+from django.urls import path, include
```

Comment:
There is a syntax error in the import statement. The `include` function is not used in this file, so it should be removed to avoid confusion.